Doc/library/ast.rst

```diff
@@ -1254,14 +1258,14 @@ Control flow
 
 .. class:: match_case(pattern, guard, body)
 
-    A single case pattern in a ``match`` statement. ``pattern`` contains the
-    match pattern that will be used to match the subject against. Notice that
-    the meaning of the :class:`AST` nodes in this attribute have a different
-    meaning than in other places, as they represent patterns to match against.
-    The ``guard`` attribute contains an expression that will be evaluated if
-    the pattern matches the subject. If the pattern matches and the ``guard`` condition
-    is truthy, the body of the case shall be executed. ``body`` contains a list
-    of nodes to execute if the guard is truthy.
+   A single case pattern in a ``match`` statement. ``pattern`` contains the
+   match pattern that will be used to match the subject against. Notice that
+   the meaning of the :class:`AST` nodes in this attribute have a different
+   meaning than in other places, as they represent patterns to match against.
```

Comment:
This section of the docs could use some copy-editing (e.g. I don't think this sentence is needed any more now that the dedicated nodes exist), but I'd prefer to leave that to a later PR rather than trying to clean it up alongside the functional change.